docsbox/docs/views.py

```diff
@@ -168,20 +172,20 @@ def delete(self, task_id):
         If task with given id is finished get the task if
         for deleting the temp file.
         """
-        task = get_task(task_id)
-        if task and task.status == "finished":
-            tmp_file_remove_task_id = task.meta["tmp_file_remove_task"]
-            if tmp_file_remove_task_id:
-                tmp_task = get_task(tmp_file_remove_task_id)
-                if tmp_task:
-                    if tmp_task.status != "finished":
-                        tmp_task = do_task(tmp_file_remove_task_id)
-                        return tmp_task.status
-                    else:
-                        return 'finished'
-                else:
-                    abort(404, "Unknown tmp_file_remove_task_id", request)
-            else:
-                return 'finished'
-        else:
-            abort(404, "Unknown task_id", request)
\ No newline at end of file
+        # task = get_task(task_id)
```

Comment:
Should we leave this here to clutter up?

Reply:
I removed the method since I added version